### DISCUSSION TITLE
Ant Media - Cluster - issue

### DISCUSSION THREAD

**Discussion by sst5000** [ant-media/Ant-Media-Server#6447] [Q&A]
We have the following queries

We have a cluster setup facing an issue, particularly with a stream with a data channel enabled.
We are constantly facing an issue related to 'no stream available to play' or 'already playing', etc kind of issues in this regard.
We tried with origin and edge servers in the cluster setup, but both behave similarly.
Attached are screenshots for your reference for origin, edge errors, and the actual streams being published in the cluster setup
Is there a setup on Ant Media that you can provide for us, that uses the cluster instance, for us to evaluate whether the same works in that setup? If that works in that setup, what's the set of changes that we needed on our instance

**Comment by yashtandon113** [comment]
@sst5000 
Thanks for reporting the issue. 

Could you please raise the support ticket at support@antmedia.io so we can check your cluster by having the credentials? Also, please share the step-by-step reproduce scenario so we can try to replicate it and investigate it.

**Comment by sst5000** [comment]
Hi,
I have already raised a ticket for the same and shared the details.

On Thu, Jun 27, 2024 at 3:43 PM yashtandon113 ***@***.***>
wrote:

> @sst5000 <https://github.com/sst5000>
> Thanks for reporting the issue.
>
> Could you please raise the support ticket at ***@***.*** so we
> can check your cluster by having the credentials? Also, please share the
> step-by-step reproduce scenario so we can try to replicate it and
> investigate it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ant-media/Ant-Media-Server/issues/6435#issuecomment-2194305791>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUTILN6U6UU5HVZJZ6SSTHLZJPQUHAVCNFSM6AAAAABJ5OXLIOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCOJUGMYDKNZZGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Regards,
Shivani
@ http://sst5000.blogspot.in/ <http://sst5000.wordpress.com>

**Reply by amarantmeida** [reply]
Hi @sst5000 
Please share the email address with which you have created the support ticket for us to assist you further on this topic?

**Comment by burak-58** [comment]
Hi my friends, It seems that we can discuss these questions on discussions. I am converting this issue to discussion.
